test/runtests.jl

```diff
@@ -57,4 +56,8 @@ using SafeTestsets
     @time @safetestset "HomotopyContinuation Extension" begin include("extensions/homotopy_continuation.jl") end
     @time @safetestset "Structural Identifiability Extension" begin include("extensions/structural_identifiability.jl") end
     
+    ### Upstream SciML and DiffEq tests. ###
+    @time @safetestset "MTK Structure Indexing" begin include("meta/mtk_structure_indexing.jl") end   
+    @time @safetestset "MTK Problem Inputs" begin include("meta/mtk_problem_inputs.jl") end   
```

Comment:
I'd put these higher up in the listing; after we test solution accuracy maybe?

Reply:
Yeah, I will move them up there